salvador/APIData.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  apiData.swift
+//  salvador
+//
+//  Created by Geury Roustand on 06.02.23.
+//
+
+import Foundation
+
+struct APIData: Codable {
```

Comment:
Those models only need to be decoded (when fetching the API data). Thus, they could conform only to the `Decodable` protocol. `Codable` includes both `Decodable` and `Encodable`. So at the moment, they have more functionality than needed.

Reply:
Thank u for the tips.